### ISSUE TITLE
CTA / Repository / Make a copy

### ISSUE THREAD

**Issue by paulsonder** [OpenConceptLab/ocl_issues#1982]

**Comment by paulsonder** [comment]
@jamlung-ri Ready for review :)

**Comment by jamlung-ri** [comment]
@paulsonder I'm noticing some potential hiccups in the design, more related to the "Create Repository" aspects of the form, not so much the "Make a copy" part of it. Is this the right design to make those comments? E.g. "Hierarchy Meaning" might be part of its own Hierarchy Setup section.

**Comment by paulsonder** [comment]
> @paulsonder I'm noticing some potential hiccups in the design, more related to the "Create Repository" aspects of the form, not so much the "Make a copy" part of it. Is this the right design to make those comments? E.g. "Hierarchy Meaning" might be part of its own Hierarchy Setup section.

No, that ticket is [here](https://github.com/OpenConceptLab/ocl_issues/issues/1978).
